plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -87,4 +87,8 @@ private MLCommonsSettings() {}
             Setting.Property.NodeScope,
             Setting.Property.Dynamic
         );
+
+    // This setting is to enable/disable model url in model register API.
+    public static final Setting<Boolean> ML_COMMONS_ALLOW_MODEL_URL = Setting
```

Comment:
Should we change this variable name also as the settings name is also changed to: `allow_registering_model_via_url` ?

Reply:
I think that's OK, it is different from the name of setting which doesn't impact any cx experience.